os/configuration/kernel-modules-kernel-headers/index.md

```diff
@@ -7,4 +7,11 @@ layout: os-default
 ## Installing Kernel Modules that require Kernel Headers
 
 
-More information coming soon!
\ No newline at end of file
+To compile any kernel modules you first need to download the kernel headers.  The kernel headers are available in the form of a system service.  Run the below to install the kernel headers:
+
+```sh
+sudo ros service enable kernel-headers
+sudo ros service up -d kernel-headers
+```
+
+The above command will end up installing the kernel headers in `/lib/modules/$(uname -r)/build`.  This will make the kernel headers available to any container running in User Docker.  Once the headers are installed any container can be deployed that compiles a kernel module just ensure you bind mount in `/usr/src` and `/lib/modules`.
```

Comment:
I don't know much about kernel headers, but would people want to know how to use them in system-docker?